internal/probation/server_test.go

```diff
@@ -51,3 +53,21 @@ func TestHealthServer_HandleFailCheck(t *testing.T) {
 		t.Errorf("Expected 'Service Not Available', got %v", body)
 	}
 }
+
+func TestHealthServer_Start(t *testing.T) {
+	server := NewHealthServer()
+	server.Start()
+
+	response, err := http.Get("http://localhost:51031/livez")
+	if err != nil {
+		t.Error(err)
+	}
+
+	if response.StatusCode != http.StatusOK {
+		t.Errorf("Expected status code %v, got %v", http.StatusAccepted, response.StatusCode)
+	}
+
+	logrus.Infof("recevied a response from the probe server: %v", response)
+
+	server.Stop()
```

Comment:
Not experienced with go testing but do we need to make sure this server is stopped if the test terminates before this point?

Reply:
it should get cleaned up, but I'll put it in a `defer` to be sure